src/CMakeLists.txt

```diff
@@ -361,6 +363,7 @@ target_compile_options(
   perf_analyzer_unit_tests
   PRIVATE
     -Wno-write-strings
+    -fcoroutines
```

Comment:
Why is this necessary? I thought using `--std=c++20` would be enough

Reply:
Ha ha ha nope. For some reason, the C++ steering committee decided to put this behind a gate. Both are needed.

Reply:
A gotcha. Well let's not forget to add -fvector -fstring -vunordered_map -fmemory -futility

Oh wait, what's that? Those aren't needed? There's different patterns used within the same language?

/s